card/mod/Modfile

```diff
@@ -27,6 +27,5 @@ mod 'account'
 mod 'navbar'
 mod 'bootstrap'
 mod 'google_analytics'
-mod 'recaptcha'
```

Comment:
I think if we change this to gem_mod we can plausibly merge this PR, though I'm concerned about the loss of CI testing of recaptcha.  We'd need to solve that...

Reply:
Yes.

I have the other branch with more mod/X components moved to gem_mod within the card repo. Maybe that's where recaptcha should be.

It is on in default Decko, yes? I'll let you decide the correct default configuration.
Technically you just add it as a Gemfile dependency. I couldn't decide whether these should or should not be in the Modfile. It can control load order, but if you want all the gem_mod/* at the end, should be fine without them in Modfile.

Reply:
Maybe reject this PR and I add recaptcha to the other change? I will post a PR for that since it seems to be mostly working now.